app/frontend/src/search/index.js

```diff
@@ -95,11 +95,11 @@ if (document.querySelector('#vacancies-hits')) {
             container: document.querySelector('#jobs_sort_form'),
             element: '#jobs_sort',
             items: [
-                { label: 'Relevancy', value: 'Vacancy' },
-                { label: 'Newest job listing', value: 'Vacancy_publish_on_desc' },
-                { label: 'Oldest job listing', value: 'Vacancy_publish_on_asc' },
-                { label: 'Newest closing date', value: 'Vacancy_expiry_time_desc' },
-                { label: 'Oldest closing date', value: 'Vacancy_expiry_time_asc' },
+                { label: 'Most relevant', value: 'Vacancy' },
+                { label: 'Newest job listing', value: 'Vacancy_publish_on_asc' },
```

Comment:
This was already correct, wasn't it? E.g. the first record in Algolia's dashboard for the index Vacancy_publish_on_asc was published in 2018, so that is the oldest job listing.
https://www.algolia.com/apps/QM2YE0HRBW/explorer/browse/Vacancy_publish_on_asc